lib/tasks/stop_data_convert_and_import.rake

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+namespace :bus_stops do
+  task import_old_data: :environment do
+    csv = CSV.parse(File.read('old_stop_data.csv'), headers: true)
```

Comment:
I think there's a method that rolls up `CSV.parse` and `File.read` into one. `CSV.read` maybe?